src/entry.js

```diff
@@ -62,7 +62,7 @@ module.exports = (argv) => {
 
   if (!utils.isValidAppInstall(command)) {
     console.error(colors.red(
-      `Looks like your package.json is missing ${PLATFORM_PACKAGE} or you haven't run npm install yet!`
+      `Looks like your package.json is missing \`${PLATFORM_PACKAGE}\`, its version is a range, or you haven't run npm install yet!`
```

Comment:
With `its version is a range`, I feel it implies the user should expect the version of the core dependency to be a range as opposed to what we really want: exact version. Or am I interpreting it wrongly?